src/test/shell/integration/runfiles_test.sh

```diff
@@ -427,5 +427,54 @@ EOF
   expect_log_once "Runfiles must not contain middleman artifacts"
 }
 
+function test_manifest_action_reruns_on_output_base_change() {
+  CURRENT_DIRECTORY=$(pwd)
+
+  OUTPUT_BASE="${CURRENT_DIRECTORY}/test/outputs/__main__"
+  TEST_FOLDER_1="${CURRENT_DIRECTORY}/test/test1"
+  TEST_FOLDER_2="${CURRENT_DIRECTORY}/test/test2"
+
+  mkdir -p "${OUTPUT_BASE}"
+  mkdir -p "${TEST_FOLDER_1}"
+  mkdir -p "${TEST_FOLDER_2}"
```

Comment:
nit: symlinking `$TEST_FOLDER_2` to `$TEST_FOLDER_1` makes it more obvious that the two are the same

Reply:
I am copying the contents now. I think that symlinking wouldn't work as the "same workspace, same output root" case didn't result in any issues even before this change.

Reply:
I thought that Bazel doesn't resolve symlinks in the name of the workspace, but you're right -- copying is the obviously correct option.